package.json

```diff
@@ -20,7 +20,7 @@
         "jsprim": "2.0.0",
         "kang": "1.3.0",
         "imgapi-cli": "git+https://github.com/joyent/sdc-imgapi-cli.git#v2.4.1",
-        "boray": "git+https://github.com/joyent/node-buckets-mdapi.git#v0.1.0",
+        "node-uuid": "1.4.0",
```

Comment:
Why `node-uuid`? I don't see any code using that module.

Reply:
Probably more carryover cruft. Will remove.

Reply:
Done